src/Admin/Views/ProviderOrganizations/AddExisting.cshtml

```diff
@@ -0,0 +1,98 @@
+@model ProviderOrganizationViewModel
+@using Bit.SharedWeb.Utilities
+
+@{
+    ViewData["Title"] = "Add Existing Organization";
+}
+
+<h1>Add Existing Organization</h1>
+<div class="row mb-2">
+    <div class="col">
+        <form class="form-inline mb-2" method="get">
+            @Html.HiddenFor(m => m.ProviderId)
```

Comment:
Why do we need this?

Reply:
MVC needs the field to be in the form to store the data to be passed to the `POST` method.